contrib/base/_grids.scss

```diff
@@ -0,0 +1,4 @@
+$nested-grid: (
```

Comment:
Any reason to not put this in `_variables.scss`, since `$nested-grid` is just a variable?